components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -284,25 +285,54 @@ export default function PrebuildsPage(props: { project?: Project; isAdminDashboa
     );
 }
 
-export function prebuildStatusLabel(prebuild?: Prebuild) {
-    switch (prebuild?.status?.phase?.name) {
+export const prebuildDisplayProps = (prebuild: Prebuild): { className: string; label: string } => {
```

Comment:
Will it be a good place to keep mappings that can be re-used into `src/data/<source>` folder?

Reply:
@mustard-mh is this the right folder for specific utils? It seems that in `src/data` we have mostly just react queries. Happy to put it there, but maybe there is even a better place?